docs/CubesterYT/KeySimulation.md

```diff
@@ -0,0 +1,43 @@
+# Key Simulation
+
+This extension allows you to simulate Key Presses and Mouse Clicks on the Project.
+
+***
+
+<h3 id="press_key">Press Key Block <span class="heading-link">(<a href="#press_key">#</a>)</span></h3>
+
+```scratch
+press key (space v) :: #BF0000
+```
+
+Presses the selected key, press time is relative to an average key press.
```

Comment:
If you _are_ going to use 1/8th of a second for the press button, list that in the documentation instead of "time is relative to an average key press."

Reply:
Depends, but for now I will leave as is, because it still makes sense.

Reply:
@NexusKitten, I'm updating it to say 1/10.